lib/between_meals/repo.rb

```diff
@@ -27,6 +27,7 @@ def initialize(repo_path, logger)
       @repo = nil
       @bin = nil
       setup
+    # rubocop:disable Lint/RescueWithoutErrorClass
```

Comment:
This is lazy programming. Just rescue `StandardError`

Reply:
Also, disabling like this disables from here until you re-enable it... so this is extra bad. :)

Reply:
I was sure how you wanted to handle these errors, i will use StandardError